configure.ac

```diff
@@ -578,6 +578,30 @@ then
 
 fi
 
+# Variation in memory internal allocation/free behavior
+if test "x${have_freerdp2}" = "xyes"
+then
+
+    # FreeRDP 2.0.0-rc0 and older automatically free rdpBitmap and its
+    # associated data member within Bitmap_Free(), relying on the
+    # implementation-specific free handler to free only implementation-specific
+    # data. This changed in commit 2cf10cc, and implementations must now
+    # manually free all data associated with the rdpBitmap, even data which
+    # was not allocated by the implementation.
+    AC_MSG_CHECKING([whether Bitmap_Free() frees the rdpBitmap and its image data])
+    AC_EGREP_CPP([\"2.0.0-dev\"], [
+
+        #include <freerdp/version.h>
+        FREERDP_VERSION_FULL
```

Comment:
What specifically are you looking for in the header file to indicate that we're on rc1+?

Is it this `FREERDP_VERSION_FULL` string?

Reply:
I mean, I get that it's the whole section included here, I was just wondering why you chose this in particular.

Reply:
This is looking for the specific text `"2.0.0-dev"` to detect that we're on rc0.

`AC_EGREP_CPP` passes the given source through the C preprocessor (CPP) and then egrep. The C preprocessor will expand that macro to its defined value, which will match the expression `"2.0.0-dev"` if the version is 2.0.0-rc0.

Reply:
I guess I actually need to escape the periods in that regex. This will currently match `"200.0-dev"`, too...

Reply:
There.

Reply:
Ah, that makes sense.